mods/ts/languages/chrome/en.ftl

```diff
@@ -184,9 +181,9 @@ button-production-types-scroll-up-tooltip = Scroll up
 button-production-types-scroll-down-tooltip = Scroll down
 
 ## mainmenu-prerelease-notification.yaml
-label-mainmenu-prerelease-notification-prompt-title = Tiberian Sun developer preview
-label-mainmenu-prerelease-notification-prompt-text-a = This pre-alpha build of OpenRA's Tiberian Sun mod is made available
-label-mainmenu-prerelease-notification-prompt-text-b = for the community to follow development and as example for modders.
-label-mainmenu-prerelease-notification-prompt-text-c = Many features are missing or incomplete, performance has not been
-label-mainmenu-prerelease-notification-prompt-text-d = optimized, and balance will not be addressed until a future beta.
-button-mainmenu-prerelease-notification-continue = I Understand
+label-mainmenu-prerelease-notification-prompt-title = Tiberian Sun developer preview.
+label-mainmenu-prerelease-notification-prompt-text-a = This pre-alpha build of OpenRA's Tiberian Sun mod is available
+label-mainmenu-prerelease-notification-prompt-text-b = for the community to follow development and serve as an example for modders.
+label-mainmenu-prerelease-notification-prompt-text-c = Please note that many features are still missing or incomplete, performance has not been
+label-mainmenu-prerelease-notification-prompt-text-d = optimized, and that game balance will not be addressed until a future beta.
+button-mainmenu-prerelease-notification-continue = I Understand.
```

Comment:
Don't add a full stop on a button.

Would also recommend keeping the original wording. It is sufficient and means we don't need to resize the dialog it goes in.